be1-go/hub/socket_types.go

```diff
@@ -0,0 +1,62 @@
+package hub
+
+import (
+	"github.com/gorilla/websocket"
+	"sync"
+)
+
+// socket types
+const (
+	clientSocket = "client"
```

Comment:
You might want to consider using a custom type here to avoid any ambiguity

Reply:
By custom type do you mean creating a structure with a string as an attribute ?

Reply:
I meant something like

`type SocketType string` and then having package level vars like `var ClientSocket = SocketType("client")`